src/plugin/plugin.ts

```diff
@@ -534,7 +534,7 @@ export interface PurchasesOffering {
   /**
    * Offering metadata defined in RevenueCat dashboard.
    */
-  readonly metadata: Map<string, any>;
+  readonly metadata: { [key: string]: unknown };
```

Comment:
should we have an example like in react native?

Reply:
Ohh yeah, I will add that after meetings